doc/recipe/preprocessor.rst

```diff
@@ -997,9 +1001,12 @@ with the remaining entries of the ``scheme`` dictionary passed as keyword
 arguments.
 
 One package that aims to capitalize on the :ref:`support for unstructured grids
-introduced in Iris 3.2 <iris:ugrid>` is :doc:`iris-esmf-regrid:index`.
+introduced in Iris 3.2 <iris:ugrid>` is :doc:`esmf_regrid:index`.
 It aims to provide lazy regridding for structured regular and irregular grids,
```

Comment:
Are you referring to "our" definition of "irregular" (i.e., general curvilinear grids)? Yes, I think it does, since most likely some of the models [here](https://esmvaltool.dkrz.de/shared/esmvaltool/regrid_tests/recipe_tos_regrid_20240614_105846/) use that type of grid.

Reply:
Yes, I was referring to "our" definition of "irregular". I now understand that that overlaps with UGRID cubes, but our definition would also apply to cubes that do not follow the UGRID standard, so would not have a mesh, right? And then iris-esmf-regrid only supports the ones that have a `.mesh` attribute that is not `None`.

Reply:
Oh wait, looks like I mixed up those definitions. In that case thuis text makes sense as it is.

Reply:
As far as I understand, our definition of "irregular" does NOT overlap with UGRID. UGRID only refers to unstructured data (1D lat and lon with common dimension), whereas our definition of irregular corresponds to structured curvilinear data (2D lat and lon with common dimensions).

> And then iris-esmf-regrid only supports the ones that have a .mesh attribute that is not None.

This is only true for unstructured data, not for (our) irregular data.